libr/core/cmd_open.c

```diff
@@ -1216,6 +1225,76 @@ static bool desc_list_json_cb(void *user, void *data, ut32 id) {
 	return true;
 }
 
+typedef struct {
+	int fd; // input and output
+	int inc;
+	int prevfd;
+	bool picknext;
+	bool found;
+	RCore *core;
+} RCmdOpData;
+
+static bool try_next(void *_user, void *_data, ut32 id) {
```

Comment:
I'd suggest to rename this function, so that becomes more clear what its' purpose is